src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -156,7 +156,7 @@ public function renderListElement(
      */
     public function output(
         FieldDescriptionInterface $fieldDescription,
-        Template $template,
+        TemplateWrapper $template,
```

Comment:
this looks like a bc break

Reply:
Yeah, I was thinking that too. But I can't figure out why this function is be public, though.

Reply:
How about introducing a new method with the correct type hint and good visibility and deprecate this one then?

Reply:
Thanks for the warning & suggestion.